tests/DevApps/WebAppCallsMicrosoftGraph/Pages/Index.cshtml.cs

```diff
@@ -29,30 +29,37 @@ public IndexModel(ILogger<IndexModel> logger, GraphServiceClient graphServiceCli
 
         public async Task OnGet()
         {
-            var user = await _graphServiceClient.Me.GetAsync(r => 
-            r.Options.WithScopes("user.read")
-            //.WithUser(User)
-            );
-         
+            try
+            {
+                var user = await _graphServiceClient.Me.GetAsync(r =>
+                r.Options.WithScopes("user.read")
+                //.WithUser(User)
+                );
+                ViewData["name"] = user.DisplayName;
+            }
+            catch (Exception)
```

Comment:
actually no, you should not do that. as there is an attribute to catch the exception.
Did you see the `[AuthorizeForScopes(Scopes = new[] { "user.read" })]` attribute, @rayluo ?

See [Managing incremental consent and conditional access](https://github.com/AzureAD/microsoft-identity-web/wiki/Managing-incremental-consent-and-conditional-access) for details

Reply:
It is true that the sample still works as-is without this part of change.

The reason I made this change is because I ran into an exception in `await _graphServiceClient.Me.GetAsync(...)` *after* I tentatively changed the "BaseUrl" in appsettings.json from `https://graph.microsoft.com/beta` to `https://graph.microsoft.com/v1.0/me`. Presumably the `_graphServiceClient.Me.GetAsync(...)` is only expected to work with that MS Graph beta endpoint. So, this change will make the sample more robust.

Reply:
@jmprieur , if we take the context above into consideration, would you suggest wrapping this code path with exception handling, or not?